app.ts

```diff
@@ -0,0 +1,285 @@
+        let toParameter = getParameters(fromParameter);
+        let maxRecordsID = recordCount - 1;
+
+        if (toParameter === maxRecordsID) {
+            alert("You have reached the final page");
+        }
+
+        let nextAmount = toParameter - fromParameter + 1;
+        let nextCountAmount = nextAmount * nextCount;
+        fromParameter = fromParameter + nextCountAmount;
+        toParameter = fromParameter + getNoOfRows();
+
+        let end = fromParameter + getNoOfRows();
+
+        if (end > maxRecordsID) {
+            toParameter = maxRecordsID;
+            fromParameter = toParameter - getNoOfRows();
+        }
+
+        nextCount = 0;
+
+        getTable();
+    };
+
+    if (nextButton !== null) {
+        nextButton.addEventListener("click", nextDebounce(next, 500));
+    }
+}
+
+// Previous
+{
+    let prevCount = 0;
+    const prevButton: HTMLElement | null = document.getElementById("prev");
+
+    const prevDebounce = (fn: any, delay: number) => {
+        return function () {
+            prevCount++;
+            clearTimeout(timer);
+            timer = setTimeout(() => {
+                fn();
+            }, delay);
+        };
+    };
+
+    let prev = () => {
+        let toParameter = getParameters(fromParameter);
+
+        if (fromParameter === 0) {
+            alert("You Are On The First Page");
+        } else {
+            let prevAmount = toParameter - fromParameter + 1;
+            let prevCountAmount = prevAmount * prevCount;
+
+            let intOne = fromParameter - prevCountAmount;
+
+            if (intOne < 0) {
+                fromParameter = 0;
+            } else {
+                fromParameter = intOne;
+            }
+
+            prevCount = 0;
+
+            getTable();
+        }
+    };
+
+    if (prevButton !== null) {
+        prevButton.addEventListener("click", prevDebounce(prev, 500));
+    }
+}
+
+// ID Jump
+{
+    const input: any = document.querySelector("input");
+
+    let idJump = () => {
+        let toParameter = getParameters(fromParameter);
+        let currentID = fromParameter;
+        let search = input.value;
+        let end = parseInt(search) + getNoOfRows();
+        let maxRecordsID = recordCount - 1;
+
+        if (search !== NaN && search !== "" && search <= maxRecordsID && search >= 0) {
+            if (end > maxRecordsID) {
+                toParameter = maxRecordsID;
+                fromParameter = toParameter - getNoOfRows();
+            } else {
+                fromParameter = parseInt(search);
+                toParameter = fromParameter + getNoOfRows();
+            }
+        } else if (search !== "") {
+            alert("Make Sure Your Desired ID Is Not A Negative Number Or Doesn't Exceed 999999");
+            fromParameter = currentID;
+            toParameter = fromParameter + getNoOfRows();
+            input.value = "";
+        }
+
+        getTable();
+    };
+
+    window.addEventListener("input", debounce(idJump, 500));
+}
```

Comment:
This is a lot of logic living in global space, either move it to a class, or execute it somewhere else.